pyproject.toml

```diff
@@ -11,14 +11,17 @@ documentation = "https://docs.beamerbridge.com/"
 
 [tool.poetry.dependencies]
 python = ">=3.10,<3.11"
-web3 = "^5.24.0"
+web3 = "^6.0.0b7"
 click = "^8.0.3"
 structlog = "^22.3.0"
 python-statemachine = "^0.9.0"
 prometheus-client = "^0.16.0"
 lru-dict = "^1.1.7"
 toml = "^0.10.2"
 atomics = "^1.0.2"
+eth-ape = "^0.6.4"
+ape-solidity = "^0.6.0"
+ape-ganache = "^0.6.1"
```

Comment:
All of these three should be in dev-dependencies, not dependencies. They are not required for non-dev usage.

Reply:
moved to dev